Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: test build build_linux
+
+test:
+		go test ./...
+
```

Comment:
Can we rely on vendor mechanism to get all dependencies locally

Reply:
Done

Reply:
As discussed will be done in next commit